Makefile.am

```diff
@@ -122,5 +122,10 @@ upload-release: $(DIST_ARCHIVES)
 	gpg --detach-sign --local-user 'stef@thewalter.net' $<
 	scp $< $<.sig $(WEBHOST):$(WEBBASE)/releases/
 
+README:
+	cp README.md README
+
 transifex:
 	cd $(srcdir) && sh build/tx-update
+
+.PHONY: README
```

Comment:
I don't think `.PHONY` is appropriate here; it's for non-filename target as far as I understand it.

Reply:
removed